spec.html

```diff
@@ -41513,8 +41513,10 @@ <h1>Corrections and Clarifications in ECMAScript 2015 with Possible Compatibilit
 
 <emu-annex id="sec-additions-and-changes-that-introduce-incompatibilities-with-prior-editions">
   <h1>Additions and Changes That Introduce Incompatibilities with Prior Editions</h1>
-  <p><emu-xref href="#sec-tonumber-applied-to-the-string-type"></emu-xref>: In ECMAScript 2015, ToNumber applied to a String value now recognizes and converts |BinaryIntegerLiteral| and |OctalIntegerLiteral| numeric strings. In previous editions such strings were converted to *NaN*.</p>
   <p><emu-xref href="#sec-reference-specification-type"></emu-xref>: In ECMAScript 2015, Function calls are not allowed to return a Reference value.</p>
+  <p><emu-xref href="#sec-tonumber-applied-to-the-string-type"></emu-xref>: In ECMAScript 2015, ToNumber applied to a String value now recognizes and converts |BinaryIntegerLiteral| and |OctalIntegerLiteral| numeric strings. In previous editions such strings were converted to *NaN*.</p>
+  <p><emu-xref href="#sec-code-realms"></emu-xref>: In ECMAScript 2018, Template objects are canonicalized based on Parse Node (source location), instead of per Realm as in previous editions.</p>
+  <p><emu-xref href="#sec-white-space"></emu-xref> In ECMASCript 2016, Unicode 8.0.0 or higher is mandated, as opposed to ECMAScript 2015, where Unicode 5.1 is mandated. In particular, this caused U+180E (MONGOLIAN VOWEL SEPARATOR), which was in the Zs category and thus treated as whitespace in ECMAScript 2015, to be moved to the Cf category (as of Unicode 6.3.0). This will cause whitespace-sensitive methods to behave differently - consider: `"\u180e".trim().length` which would be `0` in previous editions, but `1` in ECMAScript 2016 and later.</p>
```

Comment:
```suggestion
  <p><emu-xref href="#sec-white-space"></emu-xref> In ECMASCript 2016, Unicode 8.0.0 or higher is mandated, as opposed to ECMAScript 2015 which mandated Unicode 5.1. In particular, this caused U+180E MONGOLIAN VOWEL SEPARATOR, which was in the `Space_Separator` (`Zs`) category and thus treated as whitespace in ECMAScript 2015, to be moved to the `Format` (`Cf`) category (as of Unicode 6.3.0). This causes whitespace-sensitive methods to behave differently. For example, `"\u180E".trim().length` was `0` in previous editions, but `1` in ECMAScript 2016 and later.</p>
```

Reply:
Some minor edits, including using the full Unicode category names per https://github.com/tc39/ecma262/pull/621.

Also, we should note that per https://github.com/tc39/ecma262/pull/620 the latest available Unicode version is required at any time.